File: backEnd/viacord/main/models/ProcessedSample.py
class ProcessedSample:
    def __init__(self,id, sampleId,  isWeightKnown, initialWeight, sampleDate, comments, bufferVolume):
        self.id = id;
        self.sampleId = sampleId
        self.isWeightKnown = isWeightKnown
        self.initialWeight = initialWeight
        self.sampleDate = sampleDate
        self.comments = comments
        self.bufferVolume = bufferVolume

File: frontEnd/via-cord-angular/src/app/models/processed-sample.ts
export class ProcessedSample {
    id: number
    sampleId: string;
    isWeightKnown: string;
    initialWeight: number;
    sampleDate: string;
    comments: string;
    bufferVolume: number;

    constructor(id: number, sampleId: string, isWeightKnown: string, initialWeight: number, sampleDate: string, comments: string, bufferVolume: number) {
        this.id = id;
        this.sampleId = sampleId;
        this.isWeightKnown = isWeightKnown;
        this.initialWeight = initialWeight;
        this.sampleDate = sampleDate;
        this.comments = comments;
        this.bufferVolume = bufferVolume;
    }
}